doc/man/goto-synthesizer.1

```diff
@@ -0,0 +1,40 @@
+.TH GOTO-SYNTHESIZER "1" "December 2022" "goto-synthesizer-5.59.0" "User Commands"
+.SH NAME
+goto\-synthesizer \- Synthesize and apply loop contracts of goto binaries.
+.SH SYNOPSIS
+.TP
+.B goto\-synthesizer [\-?] [\-h] [\-\-help]
+show help
+.TP
+.B goto\-synthesizer \-\-version
+show version and exit
+.TP
+.B goto\-synthesizer [options] \fIin\fR [\fIout\fR]
+perform synthesis and loop-contracts transformation
+.SH DESCRIPTION
+\fBgoto-synthesis\fR reads a GOTO binary, performs a given contracts synthesis 
```

Comment:
"contracts" -> "contract's". Or:
"performs a given contracts synthesis" -> "performs synthesis for a given contract"
(if you are the kind of pedant who believe in using the possessive only for animate objects)